link-verifier/README.md

```diff
@@ -20,12 +20,20 @@ The script will print URLs that were not accessible. For Markdown files, it will
 An allow list file contains a list of non-existent URLs used as placeholder examples in a repository.
 
 ### Example
+
 Run the script with a list of space separated names of directories to exclude. Optionally increase verbosity to print all links.
 
 ```bash
 ./links-verifier/verify-links.py --test-markdown --exclude-dirs third-party cmock --include-file-types .c .h .dox --verbose
 ```
-OR Run the script with a list of files (and/or) links that you want to test specifically. 
+
+Optionally, run the script with a custom user agent.
```

Comment:
It would be good to provide context in the statement of HTTP requests for "User agent" as the meaning is not obvious otherwise. Also, it may be good to name the option as something like `--user-agent-for-http-request` to be clear about the meaning

Reply:
I think it's clear enough from context that this is referring to the http user agent.